### DISCUSSION TITLE
Should we implement an "higher only" flag?

### DISCUSSION THREAD

**Discussion by razgraf** [sablier-labs/flow#39] [Ideas]
Should there be a flag per stream that only allows the rate-per-second to be adjusted up (as in only go higher)?

**Comment by smol-ninja** [comment]
Why do you think it is necessary?

**Reply by razgraf** [reply]
Not really necessary, just a (very small) nice to have. It's like an extra assurance e.g. an employee knowing they're only going to get raises over time?

Would there be other use-cases where this "soft guarantee" would help?

**Reply by andreivladbrg** [reply]
I think this "soft guarantee" does not exist here since all streams are cancelable

**Reply by razgraf** [reply]
True true
